pkg/azuredisk/nodeserver.go

```diff
@@ -341,6 +333,58 @@ func (d *Driver) NodeExpandVolume(ctx context.Context, req *csi.NodeExpandVolume
 	return nil, status.Error(codes.Unimplemented, fmt.Sprintf("NodeExpandVolume is not yet implemented"))
 }
 
+func (d *Driver) publishBlockVolume(req *csi.NodePublishVolumeRequest) error {
```

Comment:
is it possible to combine this `publishBlockVolume` with original `NodePublishVolume` func? there are too many common code

Reply:
done.